### DISCUSSION TITLE
我在python3.6的环境中安装了pymlivus，但文档要求python3.7.1以上，请问有什么潜在风险？

### DISCUSSION THREAD

**Discussion by zyy20191** [milvus-io/milvus#18062] [Q&A and General discussion] [answered]
之前没注意到需要python3.7.1，使用下来暂时没发现什么问题，目前已经部署生产环境，但是怕有潜在风险，所以想问问，python版本过低可能出现什么BUG吗？

**Comment by yhmo** [comment, marked as answer]
以源码为准，这里https://github.com/milvus-io/pymilvus/blob/master/setup.py 要求的是python_requires='>=3.6'，所以3.6是可以的。
文档这里写得高了点：https://milvus.io/docs/v2.0.x/install-pymilvus.md
